src/btree/bt_compact.c

```diff
@@ -334,9 +334,6 @@ __wt_compact(WT_SESSION_IMPL *session)
         WT_STAT_CONN_INCR(session, session_table_compact_skipped);
         WT_STAT_DATA_INCR(session, btree_compact_skipped);
 
-        if (session == S2C(session)->background_compact.session)
-            WT_STAT_CONN_INCR(session, background_compact_skipped);
```

Comment:
This was removed as `session_table_compact_skipped` just above already covers this. We don't necessarily need to distinguish between fg and bg compaction.